.github/workflows/build-and-push-image-test.yml

```diff
@@ -9,6 +9,14 @@ env:
   TEST_RELEASE_VERSION: release-3
 
 jobs:
+  set-test-release-version:
+    name: Set Test Release Version
+    outputs:
+      value: echo ${{ steps.step1.outputs.test }}
+    steps:
+    - id: step1
```

Comment:
Could this be something nicer than 'step1', maybe 'release-version'?

Reply:
👍